### DISCUSSION TITLE
[Learn with me] Spring microservices

### DISCUSSION THREAD

**Discussion by khanhduzz** [community/community#141688] [Repositories] [answered]
### Welcome to Matcha 🍃
I'm trying to learn and build an e-commerce shopping website with Java, Spring microservices. 

This is a open source project for learning how to interact between services in an enterprise system, how to apply the latest technologies and workflow in an e-commerce shop. I plan to build it to integrate with kafka, delivery system from third-party services, calculating tax, fee and shipping base on customer location,...

If you feel interesting and want to contribute, please take a look, and work together.

I also often update topic in the project discussion, we share the knowledge and grow up together.

### The Technologies used
- Java 21
- Spring boot 3.3.3
- Swagger
- PostgreSQL
- Bootstrap
- Docker
- Sonar cloud for checking clean code.
- ...

Github repository: https://github.com/khanhduzz/matcha

**I do this all by myself, if you feel good, please give me a star ⭐️, thank you so much!**

**Comment by queenofcorgis** [comment, marked as answer]
Hi @khanhduzz , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion like seeking stars or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
